x/trigger/keeper/keeper.go

```diff
@@ -0,0 +1,44 @@
+package keeper
+
+import (
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/provenance-io/provenance/x/trigger/types"
+)
+
+const StoreKey = types.ModuleName
```

Comment:
Probably best to not have this here and instead use the `StoreKey` defined the same way in `keys.go`.